### ISSUE TITLE
can't use - crashes

### ISSUE THREAD

**Issue by gendalv** [321freddy/even-distribution#46]
shift+c
```
The mod Even Distribution (1.0.1) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event even-distribution::inventory-cleanup (ID 200)
LuaEntity doesn't contain key x.
stack traceback:
	[C]: in function '__index'
	__even-distribution__/scripts/helpers.lua:21: in function <__even-distribution__/scripts/helpers.lua:20>
	[C]: in function 'create_entity'
	__even-distribution__/scripts/visuals.lua:77: in function 'mark'
	__even-distribution__/scripts/cleanup.lua:45: in function 'func'
	__even-distribution__/scripts/util.lua:82: in function 'distribute'
	__even-distribution__/scripts/cleanup.lua:36: in function 'func'
	__even-distribution__/scripts/helpers.lua:176: in function 'each'
	__even-distribution__/scripts/cleanup.lua:29: in function 'distributeItems'
	__even-distribution__/scripts/cleanup.lua:20: in function 'func'
	__even-distribution__/framework.lua:99: in function <__even-distribution__/framework.lua:98>
```

ctrl+right click-drag or ctrl+left click-drag
```
The mod Even Distribution (1.0.1) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event even-distribution::on_player_fast_transferred (ID 131)
LuaEntity doesn't contain key x.
stack traceback:
	[C]: in function '__index'
	__even-distribution__/scripts/helpers.lua:21: in function <__even-distribution__/scripts/helpers.lua:20>
	[C]: in function 'create_entity'
	__even-distribution__/scripts/visuals.lua:39: in function 'mark'
	__even-distribution__/scripts/drag.lua:238: in function 'onStackTransferred'
	__even-distribution__/scripts/drag.lua:212: in function 'func'
	__even-distribution__/framework.lua:99: in function <__even-distribution__/framework.lua:98>
```
mods list:
``
aai-industry
Aircraft
alien-biomes
beautiful bridge railway Cargoships
Bio Industries
Bottleneck
cargo-ships
cargo-ships-graphics
CopyPasteModules
even-distribution
EvoGUI
FARL-fixed-se
Fill4Me-fixed
flib
Flow Control
FluidMustFlow
helmod
informatron
jetpack
KingsTurret-Shields
KS Power
LogisticTrainNetwork
LtnManager
ModuleInserterEx
PickerAtheneum
PickerExtended
Robo-Charge-1x1
Solar-Lamp
stdlib
thermal-solar-power
upgrade-planner-next
Warehousing

**Comment by DarkJune0122** [comment]
![image](https://github.com/321freddy/even-distribution/assets/56516330/d11bbc03-b0e4-4753-af62-8c01a64f4d7d)
Same here.

Happens when sending fuel to the miner car, that is assigned to an path, after it used up all of it's fuel.
I suppose because fuel is being used immediately - index of an item, that code is trying to fetch, is effectively missing.

Works fine if I fuel-up the miner manually a bit via default UI, and then use even distribution on it.

**Comment by 321freddy** [comment]
Should be fixed in 2.0
